### PR TITLE
issues 19 and 21

### PR DESCRIPTION
**Pull request by ghost**
1. fixed the typo in elastic.json
2. instructions have been added to /doc/source/installations.rst

### CODE REVIEW

File: doc/source/installation.rst [hunk 1]
```diff
@@ -87,13 +87,25 @@ Run the following commands to build the indices for the mentioned functionalitie
...
 | Command                                                                                  | Required for |
 +==========================================================================================+==============+
 | ``wget https://gustav1.ux.uis.no/downloads/nordlys-v02/snapshot_2015_10.txt -P data/el`` | EL           |
 +------------------------------------------------------------------------------------------+--------------+
+
+3.4 Post download settings
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+To run Nordlys smoothly the following points are to be considered:
+
+- Create a folder named "logs" and a sub-folder named "api" in Nordlys main directory, so api can create the logs file inside and starts running at ease.
```
Comment (kbalog) [inline doc/source/installation.rst]: for the log files that'll be generated by the API.

File: doc/source/installation.rst [hunk 2]
```diff
@@ -87,13 +87,25 @@ Run the following commands to build the indices for the mentioned functionalitie
...
+- If you're using macOS be a ware to change the soft limit of maxfiles to at least 64000, so mangoDB could work properly. To check the maxfiles limit of your system the below command can be used: ::
```
Comment (kbalog) [inline doc/source/installation.rst]: MongoDB

File: doc/source/installation.rst [hunk 3]
```diff
@@ -87,13 +87,25 @@ Run the following commands to build the indices for the mentioned functionalitie
...
+
+    $ launchctl limit
+
+- The Nordlys kit has to work with Java version 8.
```
Comment (kbalog) [inline doc/source/installation.rst]: Elasticsearch requires Java version 8.

**Review by kbalog — CHANGES_REQUESTED** [review]

File: doc/source/installation.rst (hunk 1 above)
Comment (ghost) [reply doc/source/installation.rst]: OK.

File: doc/source/installation.rst (hunk 2 above)
Comment (ghost) [reply doc/source/installation.rst]: OK.

File: doc/source/installation.rst (hunk 3 above)
Comment (ghost) [reply doc/source/installation.rst]: OK.

**Review by kbalog — APPROVED** [review]

**Pull request merged**